core/templates/dev/head/pages/exploration_player/TutorCardDirective.js

```diff
@@ -180,8 +181,10 @@ oppia.directive('tutorCard', [
           };
 
           $scope.isContentAudioTranslationAvailable = function() {
-            return oppiaPlayerService
-              .isContentAudioTranslationAvailable($scope.activeCard.stateName);
+            return oppiaPlayerService.isContentAudioTranslationAvailable(
+                $scope.activeCard.stateName) ||
+              LanguageUtilService.supportsAutogeneratedAudio(
```

Comment:
I don't think the directive should be caring about this level of detail. Can we subsume the dichotomous check into oppiaPlayerService.isContentAudioTranslationAvailable() (and maybe even push it further back)?

Reply:
moved back into oppia player service